### DISCUSSION TITLE
Question - warranty lost when creating additional northbound account?

### DISCUSSION THREAD

**Discussion by StefanLobbenmeierObjego** [tijsverkoyen/HomeAssistant-FusionSolar#212] [Q&A] [answered]
I followed this part of the guide:

[If you know your installer](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar#if-you-know-your-installer)

My installer got back to me and said that doing this would lose your device warranty (there is 10 years of warranty). The reason he gave was that Huawei does not want people to read this data automatically because they also sell car chargers and would rather sell you their charger than you automating a third party charger for that.

Has anyone else had a similar experience to this?

**Comment by StefanLobbenmeierObjego** [comment]
I found one more person that also had this issue in germany, maybe they will respond with details:
https://forum.huawei.com/enterprise/en/verschlechterung-der-em-garantie-em-bei-installateurszugang/thread/796259507649732608-667213868771979264

**Comment by tijsverkoyen** [comment]
Or just contact Huawei Fusion Solar Support (eu_inverter_support@huawei.com) directly and ask them if it is correct.

**Reply by StefanLobbenmeierObjego** [reply]
I just wrote to them and will make sure to update here when I get a reply

**Reply by StefanLobbenmeierObjego** [reply, marked as answer]
Got the response, we should be good (I also asked for modbus to cover my bases if I need to use the other project: https://github.com/wlcrs/huawei_solar):

Dear customer,
 
Please be wary that by using and activating a Northbound API account you do not lose the warranty of the inverter.
 
The Kiosk mode and the MODBUS mode are safe and do not void the warranty of any inverters.
Please check also the document attached and at page number 4 you can see the conditions that void the warranty.
 
Thank you for your understanding!
I wish you a peaceful weekend!
